.travis.yml

```diff
@@ -1,31 +1,63 @@
 language: ruby
+
```

Comment:
You seem to have introduced unnecessary space here, and also in other places.  Can you please clean them up?